src/boot/api.js

```diff
@@ -1,5 +1,6 @@
 import { Api, JsonRpc } from 'eosjs'
 import axios from 'axios'
+import { Screen } from 'quasar'
 
 const signTransaction = async function (actions) {
```

Comment:
@evgenibir signTransaction should accept `options` as second parameter so it could be easily overwritten if needed

Reply:
fixed